docs/distributions/manjaro/installation.md

```diff
@@ -24,45 +24,17 @@
 4. Once Secure boot is diabled, fully shutdown your Computer and Power it on again whilst holding the Option (⌥) key.
 5. Select the yellow EFI System option using the arrow keys and hit enter.
 6. Scroll down to `Boot x86 64 {Your Edition}` using the arrow keys and press enter.
-7. After booting into Manjaro, open a terminal window and run the following commands. If asked to enter a password, use `manjaro` as the password.
-
-    ```sh
-    systemctl start systemd-timesyncd.service
-    sudo sed -i 's/https:\/\/jacobpyke.xyz/https:\/\/mbp-repo.jacobpyke.xyz/' /etc/pacman.conf
-    sudo pacman -R calamares
-    sudo pacman -Sy calamares-mbp
-    ```
-
-    In case you get the error 'Calamares initialisation failed', turn off your Mac and follow steps 5 and 6 again. Then open a terminal window and run these commands instead:
-
-    ```sh
-    systemctl start systemd-timesyncd.service
-    sudo sed -i 's/https:\/\/jacobpyke.xyz/https:\/\/mbp-repo.jacobpyke.xyz/' /etc/pacman.conf
-    sudo pacman -R calamares
-    sudo pacman -Sy cmake extra-cmake-modules pkgconfig
-    sudo pacman -Sy lib32-glibc gcc
-    wget https://github.com/KDE/kpmcore/archive/v4.2.0.tar.gz
-    tar -xvf v4.2.0.tar.gz
-    cd kpmcore-4.2.0
-    mkdir build
-    cd build
-    cmake ..
-    sudo make install
-    sudo pacman -Sy calamares-mbp
-    sudo calamares
-    ```
-
-8. Open the installer and proceed normally until you arrive at the partitioning stage (the Installer will automatically start if you have used the second set of commands given above).
-9. Click Manual Partitioning.
-10. Click on `/dev/nvme0n1p1`, then press edit at the bottom of the install window, change the mount point to `/boot/efi`, and then click ok.
-11. Usually, the macOS partition is mounted to `/dev/nvme0n1p2` (the `Apple APFS` Partition). Ignore this partition.
-12. Delete the partition you created before - this is usually mounted to `/dev/nvme0n1p3`.
-13. These next steps involve partitioning the `/boot`(boot), `/`(Root) and `/home`(Home) partitions of your Linux filesystem. If you know what you're doing, feel free to skip to step 15.
+7. Open the installer and proceed normally until you arrive at the partitioning stage (the Installer will automatically start if you have used the second set of commands given above).
+8. Click Manual Partitioning.
+9. Click on `/dev/nvme0n1p1`, then press edit at the bottom of the install window, change the mount point to `/boot/efi`, and then click ok.
+10. Usually, the macOS partition is mounted to `/dev/nvme0n1p2` (the `Apple APFS` Partition). Ignore this partition.
```

Comment:
The partition is located at nvme0n1p2, but it wouldn't be "mounted" there, it shouldn't appear in ‘mount‘ as a mountpoint (although if this is just the terminology that the manjaro installer uses when referring to /dev/sda, /dev/nvme0n1p1 etc, then it should be fine).